samtranslator/schema/schema.json

```diff
@@ -199262,14 +199260,9 @@
       "additionalProperties": false,
       "properties": {
         "AccessLogSetting": {
-          "allOf": [
-            {
-              "$ref": "#/definitions/PassThroughProp"
-            }
-          ],
-          "description": "Configures Access Log Setting for a stage\\.  \n*Type*: [AccessLogSetting](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html#cfn-apigateway-stage-accesslogsetting)  \n*Required*: No  \n*AWS CloudFormation compatibility*: This property is passed directly to the [`AccessLogSetting`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html#cfn-apigateway-stage-accesslogsetting) property of an `AWS::ApiGateway::Stage` resource\\.",
+          "$ref": "#/definitions/AWS::ApiGateway::Stage.AccessLogSetting",
           "markdownDescription": "Configures Access Log Setting for a stage\\.  \n*Type*: [AccessLogSetting](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html#cfn-apigateway-stage-accesslogsetting)  \n*Required*: No  \n*AWS CloudFormation compatibility*: This property is passed directly to the [`AccessLogSetting`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html#cfn-apigateway-stage-accesslogsetting) property of an `AWS::ApiGateway::Stage` resource\\.",
-          "title": "AccessLogSetting"
+          "title": "Accesslogsetting"
```

Comment:
is this change expected?

Reply:
Yeah. Pydantic for some reason doesn't properly capitalize the title it generates. And we wanna keep the title from SAM as the name (hence `title`) of the CloudFormation property could differ.

I'll check this afterward, considering getting rid of all `title`s as they don't serve much purpose currently; they're supposed to be 1:1 the property name.